build-parent/pom.xml

```diff
@@ -32,7 +32,7 @@
         <version.buildhelper.plugin>3.6.0</version.buildhelper.plugin>
         <version.buildnumber.plugin>3.2.1</version.buildnumber.plugin>
         <version.failsafe.plugin>${version.maven.surefire.plugin}</version.failsafe.plugin>
-        <version.fermenter>2.10.3</version.fermenter>
+        <version.fermenter>2.10.5</version.fermenter>
```

Comment:
I: This version of fermenter will be released prior to merging

Reply:
Release: https://github.com/TechnologyBrewery/fermenter/releases/tag/root-2.10.5